test/jest/__tests__/interest/form.spec.ts

```diff
@@ -0,0 +1,27 @@
+import {describe, expect,it} from '@jest/globals'
+import {installQuasarPlugin} from '@quasar/quasar-app-extension-testing-unit-jest'
+import {mount} from '@vue/test-utils'
+import Form from 'components/interest/Form.vue'
+import constants from 'src/logic/constants'
+
+installQuasarPlugin()
+
+
+describe('Interest form',()=>{
```

Comment:
Could we also test if the submit button is disabled for correct cases?

Reply:
sure thing